tests/unit/backend/wmg/api/test_v2.py

```diff
@@ -91,14 +91,13 @@ def generate_expected_expression_summary_dictionary(genes, tissues, cell_types,
     return result
 
 
-def generate_test_inputs_and_expected_outputs(genes, tissues, organism, dim_size, me, expected_count, compare_dim=None):
```

Comment:
same here, would be nice to add `*`, though for functions used only in tests, it's really only beneficial for yourself so that you're forced to change all the things within your tests that call into it 😅